imperial.js

```diff
@@ -113,30 +113,39 @@ export default class Imperial {
   }
 
   tick(action) {
-    if (action.type === "playerSeating") {
-      this.seatPlayers(action);
-    } else if (action.type === "assignStartingNation") {
-      this.assignStartingNation(action);
-    } else if (action.type === "startFirstRound") {
-      this.startFirstRound();
-    } else if (action.type === "bondPurchase") {
-      this.purchaseBond(action);
-    } else if (action.type === "rondel") {
-      this.moveOnRondel(action);
-    } else if (action.type === "import") {
-      this.import(action);
-    } else if (action.type === "buildFactory") {
-      this.buildFactory(action);
-    } else if (action.type === "maneuver") {
-      const maneuverActions = this.log.filter(
-        (action) =>
-          action.type === "rondel" &&
-          (action.payload.slot === "maneuver1" ||
-            action.payload.slot === "maneuver2")
-      );
-      const lastManeuver = maneuverActions[maneuverActions.length - 1];
-      this.provinces.get(action.payload.destination).flag =
-        lastManeuver.payload.nation;
+    switch (action.type) {
```

Comment:
Do you think we may ultimately want something other than a `switch` here? I'm not sure what would be better but I guess if we want to do some OO stuff we could embellish `action`s with their own `doTheThing` methods? I don't necessarily have any opposition to the `switch` statement, I just want to be more aware of the tradeoffs.